FileShareClient/FileShareApiClient.cs

```diff
@@ -18,6 +18,7 @@ public interface IFileShareApiClient
     {
         Task<BatchStatusResponse> GetBatchStatusAsync(string batchId);
         Task<BatchSearchResponse> Search(string searchQuery, int? pageSize = null, int? start = null);
+        Task<IResult<BatchSearchResponse>> Search(string searchQuery, CancellationToken cancellationToken, int? pageSize = null, int? start = null);
```

Comment:
Please make the optional parameters mandatory
```suggestion
        Task<IResult<BatchSearchResponse>> Search(string searchQuery, CancellationToken cancellationToken, int? pageSize, int? start);
```

Reply:
resolved